etc/update-spec-tests.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
This is copied from the driver, with the exception of the path being changed from `src/test` to `tests`